Mathlib/Algebra/Order/BigOperators/Ring/Finset.lean

```diff
@@ -207,3 +207,61 @@ lemma IsAbsoluteValue.map_prod [CommSemiring R] [Nontrivial R] [LinearOrderedCom
 #align is_absolute_value.map_prod IsAbsoluteValue.map_prod
 
 end AbsoluteValue
+
+namespace Mathlib.Meta.Positivity
+open Qq Lean Meta Finset
+
+private alias ⟨_, prod_ne_zero⟩ := prod_ne_zero_iff
+
+/-- The `positivity` extension which proves that `∏ i in s, f i` is nonnegative if `f` is, and
```

Comment:
(Lean 3 copypasta?)
```suggestion
/-- The `positivity` extension which proves that `∀ i, f i` is nonnegative if `f` is, and
```

Reply:
Ah, certainly not! You are confusing `Π` and `∏`

Reply:
Ah, right... 😆